stdlib/builtins.pyi

```diff
@@ -1066,35 +1066,35 @@ def getattr(__o: object, __name: str, __default: None) -> Any | None: ...
 def getattr(__o: object, __name: str, __default: bool) -> Any | bool: ...
 @overload
 def getattr(__o: object, __name: str, __default: _T) -> Any | _T: ...
-def globals() -> dict[str, Any]: ...
+def globals() -> dict[str, object]: ...
```

Comment:
This should remain as `Any`  for the same reason we use `Any` return types instead of unions: Using `globals()` becomes quite inconvenient if you need to `assert` or `cast` every use.

Reply:
Hmm, I sort of disagree! It feels to me that using `globals()` fundamentally isn't type-safe, so it probably *should* require an explicit `cast` or `assert` if you want to use it in a type-safe way. But, I'm reverting it for now to reduce the mypy_primer diff 🙂